api-reference/50 Common/Object Structures/ExcelExportPivotGridProps/mergeColumnFieldValues.md

```diff
@@ -5,7 +5,13 @@ default: true
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies whether to merge neighbouring cells in the column field if they have the same values.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+The following image shows the exported pivot grid if you set this property to **true** or leave it unspecified:
```

Comment:
```suggestion
The following images show the exported pivot grid if you set this property to **true** or leave it unspecified
```

Reply:
Hello, @dirk-pieterse. Tech writers recently had a discussion and we decided not to divide a single sentence with images (or code blocks). It is better to create an unordered list or use some kind of the following text: 
To do A, assign B to the property: (image)
If you need C, set the property to D.

Also I'll write a message in English for you and Armina to sum up all our thoughts on this topic.

Reply:
Ok